client/static/coderogue.css

```diff
@@ -34,6 +34,12 @@ span {
 .table tbody tr {
     height: 1.2rem;
 }
+#players tr:not(:nth-child(1)) {
+    cursor: pointer;
+}
+#players tr:not(:nth-child(1)):hover {
+    color: blue;
```

Comment:
Strange. On my machine (running Chrome), the cursor change works great, but the hover effect shows as a gray highlight on all rows (including the header).

Reply:
Oh I actually pushed another change to use bootstrap's built-in class `table-hover` directly in the `<table> element` in `index.html`. You can probably see it in index.html (line 58) if you go to the `Files changed` tab of this PR.

Sorry I didn't notice the table header hover effect. Will take a look tonight when I get to the machine that has the setup to run the application.